automation/repeated_test.sh

```diff
@@ -254,7 +254,9 @@ ginkgo_params="$ginkgo_params -no-color -succinct --label-filter=${label_filter}
 if [[ -n ${NEW_TESTS} ]]; then
     readarray -t test_names <<<"$(jq -r '.[]' "${NEW_TESTS}")"
     for test_name in "${test_names[@]}"; do
-        ginkgo_params+=" -focus='${test_name}'"
+        echo "test name: ${test_name}"
+        escaped=$(echo "${test_name}" | sed -E 's/([][])/\\\1/g')
```

Comment:
Thanks a lot @akalenyu! Great stuff!

nit: this sed command is pretty cryptic :) WDYT about adding a one-line comment to explain its purpose?

Reply:
sure-done

Reply:
Thank you!

/lgtm